src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -350,6 +350,13 @@ services.AddOpenTelemetry()
         }));
 ```
 
+> [!NOTE]
+> `DefaultRequestHeaders` can be used for [HTTP Basic Access
+Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication), for
+more complex authentication requirement, `DelegatingHandler` can be used to
```

Comment:
nit: link to https://learn.microsoft.com/dotnet/api/system.net.http.delegatinghandler